github-extension/src/common/github-service.ts

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
```

Comment:
Copyright should be only 2018 as project is only here since 2018

Reply:
fixed